samples/drivers/display/sample.yaml

```diff
@@ -176,14 +170,18 @@ tests:
     harness_config:
       fixture: fixture_display
   sample.display.rk043fn66hs_ctg:
-    platform_allow: mimxrt1060_evk
+    platform_allow:
+      - mimxrt1060_evk
+      - mimxrt1050_evk
```

Comment:
is this supposed to be 1040 or 1050

Reply:
Should be 1040 on both, I missed this change- thanks